lddwrap/__init__.py

```diff
@@ -162,6 +163,9 @@ def _parse_line(line: str) -> Optional[Dependency]:
             soname=soname, path=dep_path, found=found, mem_address=mem_address)
     else:
         if len(parts) != 2:
+            if no_version_info:
```

Comment:
```suggestion
            # Please see https://github.com/Parquery/pylddwrap/pull/14
            if 'no version information available' in line:
```

(Since the variable is only used here, there is no need to introduce it specifically.)

Reply:
Heh, I coded this first and thought the other fit nicely.  Happy to change it back.